main.js

```diff
@@ -207,4 +207,4 @@ var broadcast = (message) => sockets.forEach(socket => write(socket, message));
 
 connectToPeers(initialPeers);
 initHttpServer();
-initP2PServer();
```

Comment:
Maybe this could be excluded from the PR since it's not part of the functionality